app/src/main/java/vn/sunasterisk/music_70/data/remote/TrackDataSource.kt

```diff
@@ -0,0 +1,18 @@
+package vn.sunasterisk.music_70.data.remote
+
+import vn.sunasterisk.music_70.data.model.Track
+import java.lang.Exception
+
+interface TrackDataSource {
+    interface TrackCallback{
```

Comment:
Đoạn này t nghĩ là dùng generic type để tái sử dụng code cao hơn 
```suggestion
   interface OnDataLoadedCallback<T> {
       fun onSuccess(data: T)
       fun onFailed(exception: Exception)
   }
```